defaults.yml

```
@@ -1027,3 +1027,36 @@ categories:
           - value: false
             text: '"Application Downloaded from Internet" popup will not display'
         versions: [Catalina]
+      - key: ApplePressAndHoldEnabled
+        domain: NSGlobalDomain
+        title: Choose the behavior of a key held down
```

Shorter title suggestion
```suggestion
        title: Key held down behavior
```